pyvista/core/pointset.py

```diff
@@ -818,6 +827,76 @@ def faces(self, faces):
             # TODO: faster to mutate in-place if array is same size?
             self.SetPolys(CellArray(faces))
 
+    @property
+    def regular_faces(self) -> np.ndarray:
+        """Return a face array of point indices when all faces have the same size.
+
+        Returns
+        -------
+        numpy.ndarray
+            (n_faces, face_size) Array of face indices
+
+        See Also
+        --------
+        pyvista.PolyData.faces
+
+        Notes
+        -----
+        This property does not validate that the mesh's faces are all
+        actually the same size. If they're not, this property may either
+        raise a `ValueError` or silently return an incorrect array.
+
+        Examples
+        --------
+        Get the face array of a tetrahedron as a 4x3 array
+        >>> import pyvista as pv
+        >>> tetra = pv.Tetrahedron()
+        >>> tetra.regular_faces
+        array([[0, 1, 2],
+               [1, 3, 2],
+               [0, 2, 3],
+               [0, 3, 1]])
+        """
+        return _get_regular_cells(self.GetPolys())
+
+    @regular_faces.setter
+    def regular_faces(self, faces: Union[List[List[int]], np.ndarray]):
+        """Set the face cells from an (n_faces, face_size) array."""
+        self.faces = CellArray.from_regular_cells(faces, deep=True)
+
+    @staticmethod
+    def from_regular_faces(points, faces: Union[np.ndarray, Sequence[Sequence[int]]], deep=True) -> 'PolyData':
```

Comment:
I don't have time for a review right now, but this should be a classmethod:
```py
@classmethod
def from regular_faces(cls, ...):
    poly = cls()
    ...
    return poly
```

Reply:
you're right, thanks!